target/linux/generic/patches-4.4/308-mips32r2_tune.patch

```diff
@@ -1,17 +0,0 @@
-Add -mtune=34kc to MIPS CFLAGS when building for mips32r2
```

Comment:
Instead of removing this patch, just disable CONFIG_CPU_MIPS32_R2 in the ase kernel config